GPCS4/SceModules/SceLibkernel/sce_kernel_memory.h

```diff
@@ -34,8 +34,9 @@
 // this should be decreased as direct memory being alloced
 // but currently we use fixed size
 // 
-// always 2GB
-#define  SCE_KERNEL_MAIN_DMEM_SIZE (1024 * 1024 * 1024 * 2ULL)
+// always 0x180000000 bytes
```

Comment:
why use this special value? better to add a comment

Reply:
2gb too small for persona 5 (it tryes to alloc negative size region in this case). I've got the value from